serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-opnapi-services-with-wiremock.adoc

```diff
@@ -205,19 +217,20 @@ class CountryServiceWorkflowTest {
 <1> Uses the `RestCountriesMockServer` class as a test resource.
 --
 
-+
---
-For more information about `QuarkusTestResource`, see {quarkus_test_resource_url}[Starting services before the Quarkus application starts].
---
-
-=== Starting a WireMock server to be used in a specific test
+[[proc-start-wiremock-server-specific-test]]
+=== Starting WireMock server for a specific test
 
-The following procedure describes how to start a WireMock server to be used in a specific test.
+You can also start the WireMock server to be used in a specific server.
 
-.Procedure
+.Prerequisites
+* A Serverless Workflow project is created. 
++
+For more information about creating a workflow project, see link:{create-your-first-workflow-service}[Creating your first Serverless Workflow service].
```

Comment:
Actually they won't.
Since the variable includes `xref`, you're effectively doing:

```adoc
see link:xref:getting-started/create-your-first-workflow-service.adoc
```

Which is rendered as:

```html
<a href="xref:getting-started/getting-familiar-with-our-tooling.adoc">Getting familiar with our tooling</a>
```

And that doesn't work to open that page.

Reply:
@hbelmiro Yeah, I just checked that. I see you've created the variables with xref. so, I'll create another PR and fix this. 

Thank you